src/jackdaw/test/serde.clj

```diff
@@ -0,0 +1,168 @@
+(ns jackdaw.test.serde
+  (:require
+   [clojure.tools.logging :as log]
+   [jackdaw.serdes.avro :as avro-serde]
+   [jackdaw.serdes.edn :as edn-serde]
+   [jackdaw.serdes.json :as json-serde])
+  (:import
+   (org.apache.kafka.clients.consumer ConsumerRecord)
+   (org.apache.kafka.common.serialization Deserializer Serdes Serializer
+                                          ByteArraySerializer
+                                          ByteArrayDeserializer)
+   (org.apache.kafka.common.errors SerializationException)))
+
+
+;; Access to various serdes
```

Comment:
Need to find a better home for the below section maybe (resolving serdes for topics)

Reply:
Simplified it a bit now (removed all the Avro lookup support as it as wasn't used), this is ok to leave in I think - or maybe as its only used in the tests for the test machine, it can move under the /test folder